modules/sql-engine/src/integrationTest/sql/types/time/time_parsing.test

```diff
@@ -0,0 +1,40 @@
+# name: test/sql/types/time/time_parsing.test
+# description: Test time parsing
+# group: [time]
+# note: according to SQL standard default TIME precision is 0 (Sql 2011 Part 2 Section 6.1 syntax rule 34)
+
+statement ok
+PRAGMA enable_verification
+
+query I
+SELECT '14:42:04'::TIME::VARCHAR
+----
+14:42:04
+
+# milliseconds
+query I
+SELECT '14:42:04.35'::TIME(2)::VARCHAR
+----
+14:42:04.35
+
+query I
+SELECT '14:42:04.999'::TIME(3)::VARCHAR
+----
+14:42:04.999
+
+query I
+SELECT '14:42:04.999'::TIME(6)::VARCHAR
+----
+14:42:04.999000
+
+# microseconds are not supported yet, result is rounded-up
+query I
+SELECT '14:42:04.9995'::TIME(6)
+----
+14:42:05
+
+# zeros are not trimmed
+query I
+SELECT '14:42:04.9995'::TIME(6)::VARCHAR
```

Comment:
and probably this ?

Reply:
Yes, I missed those tests, fixed, thanks.